packages/expo-video/android/build.gradle

```
@@ -1,15 +1,22 @@
 apply plugin: 'com.android.library'
 apply plugin: 'kotlin-android'
 apply plugin: 'maven-publish'
-apply plugin: 'kotlin-kapt'
 
-buildscript {
-  def expoModulesCorePlugin = new File(project(":expo-modules-core").projectDir.absolutePath, "ExpoModulesCorePlugin.gradle")
-  if (expoModulesCorePlugin.exists()) {
-    apply from: expoModulesCorePlugin
-    applyKotlinExpoModulesCorePlugin()
+group = 'host.exp.exponent'
+version = '0.15.0'
```

```suggestion
version = '0.3.1'
```
It doesn't matter, but to align with version defined below